### DISCUSSION TITLE
Problem with inputs value

### DISCUSSION THREAD

**Discussion by reziamini** [livewire/livewire#3354] [7. Bugs (archived)]
There is a problem when we are trying to update an input.
Some of the input character values will be deleted when we update the value of others' input.
Take a look at this video : 

https://user-images.githubusercontent.com/29504334/126029127-80988558-e764-4f47-9992-e507a75b8290.mp4

We had 6 characters in the second input and when we updated the next value (Which was a checkbox) 2 character of the second input was deleted.
I think it is an issue with pulling new data, and It appears when we have an internet speed issue.
Because this problem appears in the production and with local developing, we can't feel that.

I suppose that minimizing the size of pulled data will fix it.

**Comment by aliwesome** [comment]
This problem is also mentioned in https://github.com/livewire/livewire/issues/1990
